src/cli/cli.go

```diff
@@ -55,8 +56,22 @@ func preRun(cc *cobra.Command, args []string) error {
 	avoidTheseCommands := []string{
 		"corso", "env", "help", "backup", "details", "list", "restore", "delete", "repo", "init", "connect",
 	}
+
 	if len(logger.LogFile) > 0 && !slices.Contains(avoidTheseCommands, cc.Use) {
-		print.Info(cc.Context(), "Logging to file: "+logger.LogFile)
+		print.Info(ctx, "Logging to file: "+logger.LogFile)
+	}
+
+	avoidTheseCommands = []string{
+		"help", "init",
+	}
+
+	if !slices.Contains(avoidTheseCommands, cc.Use) {
+		s, acct, repoid, err := config.GetStorageRepoAndAccount(ctx)
+		if err != nil {
+			log.Debug(ctx, "Error while triggering event for ", cc.Use)
```

Comment:
This should exit by returning an error, not log.  It means we're unable to retrieve the configuration, which makes the whole command un-runnable.

Also, `log.Debug()` doesn't take in a `ctx`.  Just a message.  The context is already embedded by `logger.Ctx(ctx)` above.  `log.Debug(ctx)` Will use the context struct as the log message.